index.html

```diff
@@ -2249,6 +2244,43 @@ <h3>
           available underneath an icon).
         </p>
       </section>
+      <section>
+        <h3>
+          Launching a web application
+        </h3>
+        <p>
+          At the discretion of the operating system and/or user agent, run the
+          steps to [=launch a web application=] with a
+          [=Document/processed manifest=].
+        </p>
+        <p class="note">
+          This would typically take place when the user selects an [=installed=]
+          web app from an app launching UI surface e.g., a home screen, launcher
+          or start menu.
+        </p>
+        <p>
+          The steps to <dfn data-lt="launching a web application">launch a web
+          application</dfn> is given by the following algorithm. The algorithm
+          takes a [=Document/processed manifest=] |manifest:processed manifest|,
+          an optional |target URL:URL| and an optional
+          <a data-cite="html#post-resource">POST resource</a> |POST resource|.
+        </p>
+        <ol class="algorithm">
+          <li>If |target URL| was not given, set |target URL| to [=start URL=].
+          </li>
+          <li>Let |traversable| be the result of
+              <a data-cite="html#create-a-fresh-top-level-traversable">
+              creating a fresh top-level traversable</a> with |target URL|
+              and |POST resource| (or null if |POST resource| was not given).
+          </li>
+          <li>Let |browsing context| be the |traversable|'s
+              <a data-cite="html#nav-bc">active browsing context</a>.
+          </li>
+          <li>[=Apply=] |manifest| to |browsing context| prior to the
```

Comment:
This is hand-wavy (I know we discussed it and you wanted to write this to make sure we applied the manifest without blocking on navigation) but looking closely at the **create a fresh top level traversable** algorithm, I think it's entirely valid to just apply the manifest here without having to hand-wave about the order of navigating.

- [**create a fresh top level traversable**](https://html.spec.whatwg.org/multipage/document-sequences.html#create-a-fresh-top-level-traversable) calls **navigate** automatically.
- [**navigate**](https://html.spec.whatwg.org/multipage/browsing-the-web.html#navigate) does a bunch of pre-work (should be near-instantaneous) and then "in parallel" performs the navigation. "[in parallel](https://html.spec.whatwg.org/multipage/infrastructure.html#in-parallel)" is basically code for "queue a task to do this". So the **navigate** algorithm should return immediately after queueing the task, not actually blocking on completing the navigation.

This means if you apply the manifest here, it will be applied from the creation of the window, not blocked on navigation.

So just say "Apply _manifest_ to _browsing context_."

Reply:
Done.